### ISSUE TITLE
yii-user and yii-rbac intergration: migration issues (suggestion: enhancement)

### ISSUE THREAD

**Issue by xiandalisay** [dektrium/yii2-user#429]
Both migrations have `user` table included. When I try to migrate the second extension (either of the two), it creates an error.  Error in migrating `user` table. `user` table already exists. 

It would be better if the table creation is set "create table `user` if not extsts. It will avoid the conflict. 

Is the structure of the `user` table same for both yii-user and yii-rbac?

**Comment by dmeroff** [comment]
There is no migrations that adds `user` table in yii2-rbac. You are probably doing something wrong.

**Comment by xiandalisay** [comment]
I'd rephrase the question. Upon creating the yii template for the first time. It is needed to migrate tables. Using the console, "yii migrate" will create the `user` table. It would be better that if the `user` table exists, the migration will update the table structure.

**Comment by xiandalisay** [comment]
Actually, there is. For example, (especially for beginners). After installing the yii2 template, it is needed to invoke the command "yii migrate". This will create the `user` table.

Upon using the command, yii migrate/up --migrationPath=@vendor/dektrium/yii2-user/migrations, the error will be displayed, `user` table already exists... 

It would be better to update the column structure if the `user` table already exists. 

Apply the above migrations? (yes|no) [no]:yes
*** applying m140209_132017_init
    > create table {{%user}} ...Exception 'yii\db\Exception' with message 'SQL
ATE[42S01]: Base table or view already exists: 1050 Table 'user' already exist
The SQL being executed was: CREATE TABLE `user` (
        //columns .......
) CHARACTER SET utf8 COLLATE utf8_general_ci ENGINE=InnoDB'

in C:\wamp\www\yii2advanced2\vendor\yiisoft\yii2\db\Schema.php:615

Error Info:
Array
(
    [0] => 42S01
    [1] => 1050
    [2] => Table 'user' already exists
)

**Comment by dmeroff** [comment]
Actually there is not (if we are talking about same migrations in yii2-rbac and yii2-user, as you said in initial message). If we are talking about default application template, then yes, there is a table with same name, however you should deal with it yourself.
